specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -1475,9 +1475,13 @@
           "$ref": "#/definitions/RedirectIncompatibleRowSettings"
         },
         "logStorageSettings": {
-          "description": "Log storage settings customer need to provide when enabling session log.",
+          "description": "(Deprecated. Please use LogSettings) Log storage settings customer need to provide when enabling session log.",
```

Comment:
Is this property deprecated in ADF service? Will it break existing customers?

Reply:
No. Existing customers will still be supported with this property.